### PR TITLE
imprecv:1.0.0

### PR DESCRIPTION
**Pull request by jskherman**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: [`imprecv`](https://github.com/jskherman/imprecv) is a no-frills curriculum vitae (CV) template for [Typst](https://github.com/typst/typst) that uses a YAML file for data input in order to version control CV data easily. The template is based on the [popular résumé template on Reddit](https://web.archive.org/https://old.reddit.com/r/jobs/comments/7y8k6p/im_an_exrecruiter_for_some_of_the_top_companies/) by [u/SheetsGiggles](https://web.archive.org/https://old.reddit.com/user/SheetsGiggles) and the recommendations of the [r/EngineeringResumes wiki](https://web.archive.org/https://old.reddit.com/r/EngineeringResumes/comments/m2cc65/new_and_improved_wiki).

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/imprecv/1.0.0/utils.typ [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    else if n == 4 { month = "April" }
+    else if n == 5 { month = "May" }
+    else if n == 6 { month = "June" }
+    else if n == 7 { month = "July" }
+    else if n == 8 { month = "August" }
+    else if n == 9 { month = "September" }
+    else if n == 10 { month = "October" }
+    else if n == 11 { month = "November" }
+    else if n == 12 { month = "December" }
+    else { result = none }
```
Comment (elegaanz) [inline packages/preview/imprecv/1.0.0/utils.typ:18]: I think it should be `month` and not `result` here.

**Review by elegaanz — COMMENTED** [review]
Everything looks good, I just saw something that you might want to fix before merging.

File: packages/preview/imprecv/1.0.0/utils.typ (hunk 1 above)
Comment (jskherman) [reply packages/preview/imprecv/1.0.0/utils.typ]: I did not see that, thank you! It's now fixed.

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**